### DISCUSSION TITLE
[Feature Request]: Select support maxSelectedValues?

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#43096] [Ideas]
prettier   `<Select mode='tags' maxSelectedValues ={1}>`

**Comment by linxianxi** [comment]
```
<Select 
  value={value}
  mode='tags' 
  onChange={val => {
    if(val.length > 1) {
       message.error("........")
    }else {
       setValue(val)
    }
  }} 
/>
```

**Reply by vaynevayne** [reply]
这不还是能选多个? 
另外, 很多需求都能在用户侧做, 你的潜在意思是由用户来完成?

**Reply by linxianxi** [reply]
这怎么不行？你要不仔细看看？

**Comment by Yuiai01** [comment]
或许可以自定义一下，控制用户的选值，在超出后进行截取，可见例子： https://codesandbox.io/s/ji-ben-shi-yong-antd-5-6-1-forked-j5ydvr?file=/demo.tsx

**Comment by vaynevayne** [comment]
不用给例子了, 除了 onBlur 的问题需要 pnpm patch ,其他这些零零碎碎的, 都能自己 cover
 我说的是 feature requst : 意思是 antd 来完成这件事 而不是用户
我并没有问实现问题 qa
